android/app/src/main/res/values/strings.xml

```diff
@@ -2191,4 +2191,8 @@
 	<string name="type.amenity.events_venue">Events Venue</string>
 	<string name="type.shop.auction">Auction</string>
 	<string name="type.shop.collector">Collectables</string>
+	<string name="search_clear_button">Clear search</string>
```

Comment:
All strings are defined in data/strings/strings.txt, please check docs/TRANSLATIONS.md for details.

Check if necessary/similar strings already exist in strings.txt

Reply:
Done ^^